packages/components/src/Stepper/Stepper.component.test.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { shallow } from 'enzyme';
+import { render } from '@testing-library/react';
 import Stepper from './Stepper.component';
 
```

Comment:
may be ?
```suggestion
jest.unmock('@talend/design-system')
```

Reply:
I wasn't sure of if we want to unmock the DS in this package.
I add it so